models/requests.ts

```
@@ -84,8 +84,16 @@ export const getRequests = async (query: any) => {
     if (id) {
       const requestDoc = await requestModel.doc(id).get();
       if (!requestDoc.exists) {
+        
+        if (type === 'OOO') {
```

- why this check as we are fetching request by id and I don't think this check needed as if no request found with that particular id just throw the error not found

---

+1